documentation/milestones/individual_milestones/static_site.md

```diff
@@ -0,0 +1,199 @@
+  - most of the critical development infrastructure to be in place when we officially launch the site, so that we can deploy bug fixes or new features quickly once the site is live.
+- As an **open source contributor**, I want:
+  - the site to link to resources like the repository, chat, etc., so that I can easily learn where and how to participate in the project.
+
+## Technical description
+
+### Infrastructure Requirements
+
+The infrastructure developed to deploy and host the site should balance:
+
+- Code quality
+- Security
+- Delivery velocity
+- Cost & maintenance
+
+### User Experience Requirements
+
+The design and structure of the site should balance:
+
+- Usability
+- Accessibility
+- Site performance
+- Brand identity
+
+### Content Requirements
+
+Process for drafting and updating the content of the site should balance:
+
+- Speed & ease of content management
+- Need for review and approval
+
+### Definition of done
+
+- [ ] The following infrastructure requirements are satisfied:
+  - [ ] The code needed to build and deploy the site is merged to `main`
+  - [ ] The site is built and hosted with the tools selected in the [Front-end Planning milestone](https://github.com/HHS/grants-equity/issues/49)
+  - [ ] All code quality checks set up in the [Developer Tools milestone](https://github.com/HHS/grants-equity/issues/50) are passing
+  - [ ] The resources required to deploy and host the site are provisioned programmatically using the framework established in the [Infrastructure-as-Code milestone](https://github.com/HHS/grants-equity/issues/123)
+  - [ ] Code changes are deployed using the CI/CD pipeline set up in [the Front-end CI/CD milestone](https://github.com/HHS/grants-equity/issues/58)
+- [ ] The following user experience (UX) requirements are satisfied:
+  - [ ] Anyone can access a live version of the site at beta.grants.gov
+  - [ ] The site adopts the UI principles and framework established in the [Foundational UI milestone](https://github.com/HHS/grants-equity/issues/60)
+  - [ ] The site is 508 compliant and satisfies the other guidelines outlined in the Accessibility Planning milestone
+- [ ] The following content requirements are satisfied:
+  - [ ] The site contains information about both the NOFO Transformation and Grants.gov Modernization workstreams
+  - [ ] The content on the site has been been reviewed and approved by the relevant stakeholders within each workstream
+  - [ ] The site also links to external resources related to the project (if they are available), including:
+    - [ ] The legacy grants.gov site
+    - [ ] The open source repository
+    - [ ] The open source wiki
+    - [ ] The open source chat
+  - [ ] Stretch goal: The site provides a mechanism to collect stakeholder feedback or questions
+
+### Proposed metrics for measuring goals/value/definition of done
+
+1. Number of unique site visitors
+2. Total number of site visits
+3. Stretch: Number of form responses
```

Comment:
Other metrics to consider adding: 

* unique/total visitors to GitHub (traffic will be driven there from the beta landing page) 
* Metrics on each communication channel we end up displaying on static site (i.e., number of people who join Slack, number of people who email us, who join email listserv, etc) 
* Number of visitors who click into the PDFs of the NOFO prototypes

I'm not super attached to these, just for your consideration!

Reply:
Added these in [this commit](https://github.com/HHS/grants-equity/pull/243/commits/b1620b74ecc0d590e7fc06575fd8f47edc035bc6)